sdk/lib/debug/debug.cc

```diff
@@ -335,7 +336,13 @@ debug_log_message_write(const char          *context,
 	DebugPrinter printer;
 	printer.write("\x1b[35m");
 	printer.write(context);
+#if 0
```

Comment:
This probably wants wiring up in an option, it's mostly disabled for now so that I can do timing checks without it there (it slows things down quite a bit with a real UART).